_addons_ios/google-assistant/readme.md

```diff
@@ -15,40 +15,13 @@ With the Action you can voice control your openHAB items and it supports lights,
 
 If you have any issues, questions or an idea for additional features, please take a look at the [repository on GitHub](https://github.com/openhab/openhab-google-assistant).
 
-## Table of Contents
-
-- [Latest Changes](#latest-changes)
-- [General Configuration](#general-configuration)
-- [Item Configuration](#item-configuration)
-  - [`Switch`](#switch)
-  - [`Light`](#light)
-  - [`Light` as Group with separate Controls](#light-as-group-with-separate-controls)
-  - [`Scene`](#scene)
-  - [`Outlet`, `Coffee_Maker`, `WaterHeater`, `Fireplace`](#outlet-coffee_maker-waterheater-fireplace)
-  - [`Valve`](#valve)
-  - [`Sprinkler`, `Vacuum`](#sprinkler-vacuum)
-  - [`Lock`](#lock)
-  - [`SecuritySystem` as Switch](#securitysystem-as-switch)
-  - [`SecuritySystem` as Group with advanced functionality](#securitysystem-as-group-with-advanced-functionality)
-  - [`Camera`](#camera)
-  - [`Speaker` (volume control only)](#speaker-volume-control-only)
-  - [`TV`](#tv)
-  - [`Fan`, `Hood`, `AirPurifier`](#fan-hood-airpurifier)
-  - [`Awning`, `Blinds`, `Curtain`, `Door`, `Garage`, `Gate`, `Pergola`, `Shutter`, `Window`](#awning-blinds-curtain-door-garage-gate-pergola-shutter-window)
-  - [`Charger`](#charger)
-  - [`TemperatureSensor`](#temperaturesensor)
-  - [`Thermostat`](#thermostat)
-  - [`Sensor`](#sensor)
-- [Additional Configuration](#additional-configuration)
-  - [Two-Factor-Authentication](#two-factor-authentication)
-- [Setup \& Usage of the Google Home App](#setup--usage-of-the-google-home-app)
-- [Example Voice Commands](#example-voice-commands)
-- [Frequently Asked Question](#frequently-asked-question)
+[[toc]]
 
 ## Latest Changes
 
 ::: tip State of this document
-This documentation refers to release [v3.5.1](https://github.com/openhab/openhab-google-assistant/releases/tag/v3.5.1) of [openHAB Google Assistant](https://github.com/openhab/openhab-google-assistant) published on 2022-11-09 :::
+This documentation refers to release [v3.5.1](https://github.com/openhab/openhab-google-assistant/releases/tag/v3.5.1) of [openHAB Google Assistant](https://github.com/openhab/openhab-google-assistant) published on 2022-11-09
```

Comment:
Just out of curiousity: You are writing this by hand currently?

Reply:
Yes. Had a short look in the mentioned workflows but did not find a useful application yet.

Reply:
As I also do releases more or less manual, this effort is acceptable.

Reply:
No need to hurry.
I will make some thoughts and look for a solution that will work for all external components then. 🙂